### ISSUE TITLE
PyTorch version recommendation could be removed/bumped

### ISSUE THREAD

**Issue by AsuMagic** [pyannote/pyannote-audio#1168]
Hello,

Your README mentions the following in the install instructions, following PR #1030:

```
# pytorch 1.11 is required for speechbrain compatibility
# (see https://pytorch.org/get-started/previous-versions/#v1110)
conda install pytorch==1.11.0 torchvision==0.12.0 torchaudio==0.11.0 -c pytorch
```

I thought I'd let you know that the PyTorch `<= 1.11` requirement is outdated as of [SpeechBrain v0.5.13](https://github.com/speechbrain/speechbrain/releases/tag/v0.5.13) which is now advertised as fully compatible with PyTorch 1.12.  
Its [`requirements.txt`](https://github.com/speechbrain/speechbrain/blob/develop/requirements.txt) also does not specify an explicit maximum version anymore.

So if you want to bump (or not restrict) the recommended PyTorch version, you might be able to :)

**Comment by hbredin** [comment]
Thanks! Would you contribute this change through a PR?
This would probably also mean updating the [intro](https://github.com/pyannote/pyannote-audio/blob/develop/tutorials/intro.ipynb) notebook.

**Comment by hbredin** [comment]
Thanks for working on this. You could indeed rerun the notebook, make the PR, and, based on this PR, I'll have a look at the reason behind this huge DER...

**Comment by AsuMagic** [comment]
Closed by #1323
